nifi-commons/nifi-web-client/src/main/java/org/apache/nifi/web/client/StandardHttpUriBuilder.java

```diff
@@ -16,65 +16,171 @@
  */
 package org.apache.nifi.web.client;
 
-import okhttp3.HttpUrl;
 import org.apache.nifi.web.client.api.HttpUriBuilder;
 
 import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URLDecoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
 /**
  * Standard HTTP URI Builder based on OkHttp HttpUrl
```

Comment:
no longer OkHttp HttpUrl

Reply:
Thanks @joewitt! I pushed an update to correct the comments.